src/test/java/Homework21.java

```diff
@@ -0,0 +1,55 @@
+import io.github.bonigarcia.wdm.cache.ResolutionCache;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework21 extends BaseTest{
+
+@Test
+
+    public void renamePlaylist(){
+  //  GIVEN
+    openUrl();
+    enterEmail("andrei.bryliakov01@testpro.io");
+    enterPassword("te$t$tudent");
+    clickLoginBtn();
+  //   WHEN
+    doubleClick();
+    playListNewName();
+    click();
+  //   THEN
+    WebElement createdElementNewName = wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("[class='success show']")));
+    Assert.assertTrue(createdElementNewName.isDisplayed());
+
+}
+
+    public void doubleClick(){
+        WebElement playListElement = wait.until(ExpectedConditions.elementToBeClickable(By. cssSelector(".playlist:nth-child(3)")));
+    Actions actions = new Actions(driver);
+    actions.doubleClick(playListElement).perform();
+
+    }
+
+    public void  playListNewName() {
+      WebElement playListName = wait.until(ExpectedConditions.elementToBeClickable(By. cssSelector("input[name='name']")));
+      // playListNewName.clear();
+    playListName.sendKeys("Hello");
+    }
+
+    public void click() {
+        WebElement click = wait.until(ExpectedConditions.elementToBeClickable(By. cssSelector("[type='search']")));
+        click.click();
+    }
+
+}
+
+
+
+
+
+
+
```

Comment:
**Please delete the empty line!** An Empty Line is a "Code Smell"
The short answer is, _They’re more an annoyance than a problem.They won’t actually break anything—which is why your code continues to run—but they make it awkward to read and change your code, especially if you’re dealing with multiple line breaks per line of real code. They also add just a tiny bit of size to your files, which is nice to reduce._